lighthouse-cli/test/fixtures/offline-ready-sw.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
maybe put this into a directory under fixtures? Not a huge deal, but we're probably going to have to do it eventually